home/views.py

```diff
@@ -140,8 +144,8 @@ def moreinfo(request, name):
     salon = get_object_or_404(Salon, url=name)
     services = Services.objects.filter(salons__name=salon.name)
     products = Products.objects.filter(salons__name=salon.name)
-    comments = Comments.objects.filter(
-        salon__id=salon.id).order_by("-created_date")
+    comments = Comments.objects.filter(salon__id=salon.id)
```

Comment:
*you affected line `147` which will in turn affect sorting of comments

Reply:
Reverted to original filter by created date

Reply:
![Peek 2019-08-07 14-07](https://user-images.githubusercontent.com/27855418/62618421-d7aa5d00-b91c-11e9-825f-4856da20bfd7.gif)